{{cookiecutter.project_dirname}}/pyproject.toml

```diff
@@ -46,7 +50,7 @@ extend-exclude = [
     ".vscode*",
     "*/migrations/*",
 ]
-extend-ignore = [
+ignore = [
```

Comment:
```suggestion
extend-ignore = [
```

Reply:
`extend-ignore` è deprecato in favore di `ignore`: https://docs.astral.sh/ruff/settings/#extend-ignore